### PR TITLE
[#WPLAT-886 Bug] Fixed error handling, mongoose package version

### PR DESCRIPTION
**Pull request by tamassandor-planorama**
Ticket: https://planorama.atlassian.net/browse/WPLAT-886
- `mongoose` version fixed on request of @jeanbaptiste-brasselet-planorama 
- added proper error handling

### CODE REVIEW

**Comment by jeanbaptiste-brasselet-planorama** [comment]
all the other method were already ok ? if ok then ok !

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "supertest": "^0.15.0"
   },
   "peerDependencies": {
-    "mongoose": "^4.3.4"
+    "mongoose": "4.6.0"
```
Comment (jeanbaptiste-brasselet-planorama) [inline package.json]: There is another place in the code (devDependencies, to ensure the unit test are ok)

Comment (tamassandor-planorama) [inline package.json]: done

**Comment by tamassandor-planorama** [comment]
Checked the other methods, only place left out seems only the `getMany`.

**Comment by szabolcs-szilagyi** [comment]
ok

**Comment by jeanbaptiste-brasselet-planorama** [comment]
you need to upgrade the version of noot in the package.json file

**Comment by jeanbaptiste-brasselet-planorama** [comment]
ok

**Pull request merged**